docs/attach-compute-on-aks-v1-cluster.md

```diff
@@ -0,0 +1,64 @@
+Sure, here is a possible better version of the markdown document:
+
+# How to use an existing AKS v1 compute as a v2 Kubernetes compute (Preview)
```

Comment:
User may dont aware of what is v1 and v2. I feel the title better to be "Enable training and inference ML workloads in legacy AKSCompute"

Reply:
changed